test/on-error.js

```diff
@@ -34,8 +34,8 @@ async function main () {
     reply.from(`http://localhost:${target.server.address().port}/`,
       {
         onError: (reply, { error }) => {
-          t.equal(error.status, 504)
-          reply.code(error.status).send(error)
+          t.equal(error.statusCode, 504)
```

Comment:
I assume because I had to change this unit test, it will be a breaking change

Reply:
Is the error a `GatewayTimeoutError` instance?

Reply:
I will extend the test to check if it is a specific error.

Reply:
Changed it to check for the Error. Does this help?